### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Feichtmeier**
## Pull request checklist

- [X] this PR does not introduce visual changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,10 @@ Common flutter widgets useful for building desktop and web applications.
 ## What is this?
 
 A set of convenient widgets made with `material.dart` to easily build desktop and web applications.
+Design references are the [Ubuntu yaru gnome themes](https://github.com/ubuntu/yaru) and Canonicals [vanilla framework](https://vanillaframework.io/).
```
Comment (jpnurmi) [inline README.md:12]:
```suggestion
Design references are the [Ubuntu Yaru theme suite](https://github.com/ubuntu/yaru) and the [Vanilla framework](https://vanillaframework.io/) by Canonical.
```

File: README.md [hunk 2]
```diff
@@ -32,6 +27,6 @@ The answer is that `material.dart` brings functionally everything that one needs
 
 So `yaru_widgets.dart` uses the completeness and the level of polish `material.dart` has and adds a set of widgets to it that are useful (but not necessarily the only way) to build desktop and web applications so they all have a consistent look and use the same design patterns.
 
-## Recommended themes
+## Tested themes
 
-Themes inside [`yaru.dart`](https://github.com/ubuntu/yaru.dart) are recommended but all flutter material themes should work.
+Themes inside [`yaru.dart`](https://github.com/ubuntu/yaru.dart) are the only material themes tested with yaru_widgets and we do not currently recommend to use any other themes.
```
Comment (jpnurmi) [inline README.md:32]:
```suggestion
Themes inside [`yaru.dart`](https://github.com/ubuntu/yaru.dart) are the only Material themes tested with yaru_widgets and we do not currently recommend to use any other themes.
```

**Review by jpnurmi — APPROVED** [review]
LGTM :+1:

**Comment by Feichtmeier** [comment]
Used again the "commit suggestion" feature! :D This time it should not break any formating like within dart files xD

**Pull request merged**